reframe_config.py

```diff
@@ -185,13 +185,20 @@
                     'prefix': '%(check_system)s/%(check_partition)s',
                     'level': 'info',
                     'format': (
-                        '%(check_job_completion_time)s|reframe %(version)s|'
-                        '%(check_info)s|jobid=%(check_jobid)s|'
+                        '%(check_job_completion_time)s|'
+                        'reframe %(version)s|'
+                        '%(check_info)s|'
+                        'jobid=%(check_jobid)s|'
                         '%(check_perf_var)s=%(check_perf_value)s|'
-                        'ref=%(check_perf_ref)s '
+                        'units=%(check_perf_unit)s|'
```

Comment:
What's the rationale for replacing `%(check_perf_unit)s` with `units=%(check_perf_unit)s`?  Is it for making it easier to identify the field?  I think that's ok, but asking to understand the reason of the change.  Although, I'd keep it closer to the rest of the reference values.

Reply:
> Is it for making it easier to identify the field? 

Yep, exactly that. Happy for it to go wherever!